### PR TITLE
#918 [docs] worker introduction

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #918)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
[link](https://github.com/chia7712/ohara/blob/fix_918_4/docs/user_guide.md#worker)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/user_guide.md [hunk 1]
```diff
@@ -257,9 +257,8 @@ dataDir=/tmp/zookeeper/data
 server.0=node00:2888:3888
 ``` 
 
-Except for clientPort and server.n (see [Zookeeper APIs](rest_interface.md#zookeeper)), other options are auto-generated by
-Ohara Configurator and them all are the default value recommended by zookeeper. Feel free to file a issue to Ohara community if
-you have better configs for zookeeper.
+Most options are auto-generated by Ohara Configurator, and and [Zookeeper APIs](rest_interface.md#create-a-zookeeper-cluster)
```
Comment (saivirtue) [inline docs/user_guide.md]: - duplicate and

File: docs/user_guide.md [hunk 2]
```diff
@@ -304,13 +303,60 @@ zookeeper.connect=node00:2181
 advertised.listeners=PLAINTEXT://node00:9092
 ```
 
-Except for listeners, other configs are filled by Ohara Configurator. Ohara community always welcomes user to raise
+Most configs are filled by Ohara Configurator, and [Broker APIs](rest_interface.md#create-a-broker-cluster)
```
Comment (saivirtue) [inline docs/user_guide.md]: I think here we can use the same sentence as zookeeper :
Most options are auto-generated by Ohara Configurator, and...

Comment (chia7712) [reply docs/user_guide.md]: got it

**Comment by chia7712** [comment]
retry common

**Pull request merged**